### DISCUSSION TITLE
Does biome support formatting of markdown files?

### DISCUSSION THREAD

**Discussion by hamlim** [biomejs/biome#923] [Q&A] [answered]
Couldn't find an answer on the docs site or in issues/discussions on the repo, wondering if Biome supports formatting markdown files, and if so how does that compare to either Prettier or Dprint?

**Comment by Conaclos** [comment, marked as answer]
Hi @hamlim, Biome doesn't support Markdown formatting yet. We are currently working on CSS. We also need HTML formatting before working on Markdown.

**Reply by PaulRBerg** [reply]
Do you have an ETA for when HTML and Markdown will be supported?

**Reply by dyc3** [reply]
No, sorry we don't have any ETAs.

**Comment by ematipico** [comment]
https://biomejs.dev/internals/language-support/

**Comment by cinderblock** [comment]
https://github.com/biomejs/biome/issues/3718
